conditional-signing/browser/README.md

```diff
@@ -0,0 +1,96 @@
+   - After signing this message, you should see in the JavaScript console something similar to:
+     ```json
+     Got Auth Sig for Lit Action conditional check!
+     {
+        "sig": "0x6607f337861bf1f2ce331ee3a6ef7b5c08e31fb124e2139c48c6f0223258727306a1b38dbd9fa696f6ecc8a9dca82cca61e23882c0787bbf07dc0ba7bbb1ef081c",
+        "derivedVia": "web3.eth.personal.sign",
+        "signedMessage": "localhost wants you to sign in with your Ethereum account:\n0xA89543a7145C68E52a4D584f1ceb123605131211\n\nThis is a test statement.  You can put anything you want here.\n\nURI: http://localhost\nVersion: 1\nChain ID: 1\nNonce: 0x40e2d121e4513c45063212f65f139b300625a91e3900a76f56478ffd815bef21\nIssued At: 2024-05-03T23:43:48.500Z\nExpiration Time: 2024-05-04T23:43:45.894Z",
+        "address": "0xA89543a7145C68E52a4D584f1ceb123605131211"
+     }
+     ```
+5. After signing that message, depending on whether or not you specified a PKP Public Key in the `.env` file:
+   a. If you did, you'll be on the next step
+   b. If you didn't, you will be prompted by your wallet to sign a transaction to mint a new PKP on the Lit Chronicle rollup. This transaction will cost `testLPX` (which is only a testnet token with **no** real-world value). After signing this transaction you should see in the JavaScript console something similar to:
+   ```json
+    Minted PKP!
+    {
+        "tokenId": "0x59cb949a00d46ccd9deceb9912f935871deea7711951433254135242f53153fd",
+        "publicKey": "0400f74bb0ece6c5c4f9577f358a77790a7b790eef48e3367bd05a3cc648504fb1efc99b840549809b85d66c5f55503f9ef92eeee5e3eb30ee06976d7b5fbc3c90",
+        "ethAddress": "0x5c2e895EDAdce9fB019D497238013bF7527d6690"
+    }
+   ```
+6. Finally, if the Ethereum address you signed the SIWE message with has a balance equal to or greater than 1 Wei on Ethereum Mainnet, you should see in the JavaScript console something similar to:
+   ```json
+   litActionSignatures:
+   {
+     "claims": {},
+     "signatures": {
+       "sig": {
+         "r": "b276fc11cbb133eeb25e0d5ff5c759514cfb21f15d0248cdfafc0440fa148d2a",
+         "s": "006e3b9ece94f8531b5baa4b428a8c6155335e61a5639980d20c9d7477f6746f",
+         "recid": 1,
+         "signature": "0xb276fc11cbb133eeb25e0d5ff5c759514cfb21f15d0248cdfafc0440fa148d2a006e3b9ece94f8531b5baa4b428a8c6155335e61a5639980d20c9d7477f6746f1c",
+         "publicKey": "041E7A220A697F47491525798337BFAAC6073C6094FDDE9187D749D28D947F59FE73FBAE024FC0B87D2A61068EA8087E94ECC843820752295307537F9D06432876",
+         "dataSigned": "7D87C5EA75F7378BB701E404C50639161AF3EFF66293E9F375B5F17EB50476F4"
+       }
+     },
+     "logs": ""
+   }
+   ```
+   If the address does not have 1 or more Wei, you should see an empty signature:
```

Comment:
nit: If the balance isn't met how about returning something in the `setResponse` saying you didn't met the condition for signing?